src/test/java/org/mskcc/cbio/portal/integrationTest/dao/TestDaoCancerStudy.java

```diff
@@ -68,7 +71,7 @@ public class TestDaoCancerStudy {
 	@Test
     public void testDaoCancerStudy() throws DaoException, IOException {
 
-        assertEquals("Breast Invasive Carcinoma", DaoTypeOfCancer.getTypeOfCancerById("BRCA").getName());
+        Assert.assertEquals("Breast Invasive Carcinoma", DaoTypeOfCancer.getTypeOfCancerById("BRCA").getName());
```

Comment:
Since you are doing the static import you should be able to get rid of the Assert. part